src/twisted/python/test/_deprecatetests.py.3only

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+import inspect
+
+from twisted.python.deprecate import _passedSignature
+from twisted.trial.unittest import SynchronousTestCase
+
+
+class KeywordOnlyTests(SynchronousTestCase):
+    """
+    Keyword only arguments (PEP 3102).
+    """
+    def checkPassed(self, func, *args, **kw):
+        """
+        Test an invocation of L{passed} with the given function, arguments, and
+        keyword arguments.
+
+        @param func: A function whose argspec to pass to L{_passed}.
+        @type func: A callable.
+
+        @param args: The arguments which could be passed to L{func}.
+
+        @param kw: The keyword arguments which could be passed to L{func}.
+
+        @return: L{_passed}'s return value
+        @rtype: L{dict}
+        """
+        return _passedSignature(inspect.signature(func), args, kw)
+
+
+    def test_passedKeywordOnly(self):
+        """
+        Keyword only arguments follow varargs.
+        They are specified in PEP 3102.
+        """
+        def func1(*a, b=True):
+            """
+            b is a keyword-only argument, with a default value.
+            """
+            pass
```

Comment:
The `pass` is unnecessary here.  The docstring serves the syntactic purpose of having _something_ in the body.